src/me/StevenLawson/TotalFreedomMod/Commands/Command_tagnyan.java

```diff
@@ -29,6 +30,19 @@ public boolean run(CommandSender sender, Player sender_p, Command cmd, String co
 
         final TFM_PlayerData data = TFM_PlayerData.getPlayerData(sender_p);
         data.setTag(tag.toString());
+        
+        if (!TFM_AdminList.isSuperAdmin(sender))
+        {
+            for (String word : Command_tag.FORBIDDEN_WORDS)
+            {
+                if (args[0].toLowerCase().contains(word))
+                {
+                    playerMsg("That tag contains a forbidden word (" + args[0] + ")");
+                    data.setTag(null);
```

Comment:
I mean, there isn't a code to remove the new one but leave the pre-existing tag untouchable.
Unless you use ``` event.setCancelled(true); ``` you will need to fix everything up.